crypto/ocsp/ocsp_server.c

```diff
@@ -12,6 +12,13 @@
 
 #include "internal.h"
 
+int OCSP_request_onereq_count(OCSP_REQUEST *req) {
+  return (int)sk_OCSP_ONEREQ_num(req->tbsRequest->requestList);
+}
+
+OCSP_ONEREQ *OCSP_request_onereq_get0(OCSP_REQUEST *req, int i) {
+  return sk_OCSP_ONEREQ_value(req->tbsRequest->requestList, i);
```

Comment:
Hmmmmm might need a new macro that returns null, if you use GUARD_PTR which returns zero (which is basically null) does the compiler get mad?

Reply:
Seems like it works! Also verified that returning 0 is equivalent to NULL.